src/rbus/rbus.c

```diff
@@ -221,19 +222,36 @@ void rbusEventSubscription_free(void* p)
     free(sub);
 }
 
-static rbusEventSubscription_t* rbusEventSubscription_find(rtVector eventSubs, char const* eventName, rbusFilter_t filter)
+static rbusEventSubscription_t* rbusEventSubscription_find(rtVector eventSubs, char const* eventName,
+        rbusFilter_t filter, int32_t* interval, uint32_t* duration)
```

Comment:
no need pass pointer to interval, duration for finding it

Reply:
code changes are done.